lightning/src/ln/channelmanager.rs

```diff
@@ -1797,6 +1797,23 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 		&self.default_configuration
 	}
 
+	fn create_insert_outbound_scid_alias(&self) -> u64 {
```

Comment:
nit: create_or_insert_outbound_scid_alias

Reply:
But it does both, so its not `_or_`.

Reply:
Ah, right lol. Maybe `_and_` or drop `insert_` if it is relevant to the caller? No strong opinion.